src/main/java/org/intellij/sdk/codesync/activityAlert/AlertAction.java

```diff
@@ -0,0 +1,73 @@
+package org.intellij.sdk.codesync.activityAlert;
+
+import com.intellij.ide.BrowserUtil;
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationAction;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.project.Project;
+import org.intellij.sdk.codesync.CodeSyncLogger;
+import org.intellij.sdk.codesync.alerts.ActivityAlerts;
+import org.intellij.sdk.codesync.files.AlertsFile;
+import org.intellij.sdk.codesync.utils.CodeSyncDateUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Instant;
+import java.time.ZoneId;
+
+public class AlertAction extends NotificationAction {
+
+    String actionName;
+    Project project;
+    String email;
+    String teamActivityURL;
+
+    public AlertAction(String actionName, Project project, String email, String teamActivityURL){
+        super(actionName);
+        this.actionName = actionName;
+        this.project = project;
+        this.email = email;
+        this.teamActivityURL = teamActivityURL;
+    }
+
+    @Override
+    public void actionPerformed(@NotNull AnActionEvent e, @NotNull Notification notification) {
+
+        notification.expire();
+
+        if (actionName.equals("View Activity")) {
```

Comment:
How about making a Constant of these options since we are using at 2+ places, would be easy to change only at 1 place